src/main/java/de/tum/in/ase/notification/model/TestCase.java

```diff
@@ -23,7 +18,7 @@ public class TestCase {
 
     private List<Failure> failures;
 
-    private List<Error> errors;
+    private List<java.lang.Error> errors;
```

Comment:
```suggestion
    private List<Error> errors;
```
I think this is meant to be our own data class here.